app/models/manageiq/providers/amazon/cloud_manager.rb

```diff
@@ -47,6 +47,10 @@ class ManageIQ::Providers::Amazon::CloudManager < ManageIQ::Providers::CloudMana
   supports :regions
   supports :discovery
 
+  def ensure_network_manager
```

Comment:
@durandom right, that was being leveraged in the next PR https://github.com/ManageIQ/manageiq-providers-amazon/pull/20/files

but I am no longer sure if it needs to be divided. Too many dependencies for my brain :-)